config/application.rb

```diff
@@ -49,6 +49,10 @@ class Application < Rails::Application
     config.active_support.hash_digest_class = ::Digest::MD5 # New default is ::Digest::SHA1
     ### END FRAMEWORK DEFAULT OVERIDES
 
+    # permit using a string uuid as namespace
+    # see https://github.com/rails/rails/pull/37682
```

Comment:
it might make more sense to link to  https://guides.rubyonrails.org/configuring.html#config-active-support-use-rfc4122-namespaced-uuids 

This is probably a 7.0 default and should be commented as such?

Reply:
having said "this is probably a 7.0 default" - rather than configuring it here (where we add other default overrides) I thought about _uncommenting_ it [here]( https://github.com/forem/forem/blob/djuber/set-uuid-namespace/config/initializers/new_framework_defaults_7_0.rb#L102-L107), but we have config.load_defaults 6.0 earlier in config/application.rb so the 7.0 file is presumably not loaded/read.

Reply:
yeah, we actually haven't done the work of fully move past Rails 6 default

Reply:
Just came from https://github.com/forem/forem/pull/17371#issuecomment-1103954089, but why not follow the guide likehttps://guides.rubyonrails.org/v7.0/upgrading_ruby_on_rails.html#configure-framework-defaults ?